core/domain/contributor_admin_dashboard_services.py

```diff
@@ -233,7 +233,7 @@ def get_translation_submitter_total_stats(
     page_size: int,
     offset: int,
     language_code: str,
-    sort_by: Optional[suggestion_models.SortChoices.value],
+    sort_by: Optional[str],
```

Comment:
Re https://github.com/oppia/oppia/pull/18576#discussion_r1249593166 -- I'm not sure I buy the rationale. The type checking should always be as strict as possible, so I wouldn't suggest relaxing it.

My original comment was about something different. Here it is for reference:

> I don't think I have seen this typing before. Can you point me to somewhere else in the codebase where things are typed by {{enum}}.value?
>
> If not, please follow existing codebase conventions, or explain clearly why this is different from all the other cases.

Can you address the question here?

Reply:
I am not able to find an example where we have used {enum}.values as a typing but it is necessary here because we have great a new constant CD_ADMIN_STATS_SORT_OPTIONS in constants.ts which is mapped to the values of this enum. On using the constant value directly as typing there is a mypy error

error: Name 'constants.CD_ADMIN_STATS_SORT_OPTIONS.value' is not defined  [name-defined]